### PR TITLE
refactor/916

### LINKED ISSUE #916: Consolidate aircraft removal logic
`AircraftModel` does some of the logic and the `AircraftController` does some too,  with no clear way to communicate between the two.

consolidate the logic and make this simpler

### PR DESCRIPTION
**Pull request by n8rzz**
Resolves openscope/openscope#916 

* consolidates aircraft removal logics
* renames `AircraftModel#inside_ctr` -> `AircraftModel#isControllable`
* simplifies aircraft update logic in `AircraftController`

### CODE REVIEW

**Comment by n8rzz** [comment]
@erikquinn mad one more small change in addition to the ones you suggested 😉

File: src/assets/scripts/client/aircraft/AircraftModel.js [hunk 1]
```diff
@@ -250,7 +250,7 @@ export default class AircraftModel {
 
         /**
          * Azimuth from airport center to aircraft, in radians
-         *
+         *G
```
Comment (erikquinn) [inline src/assets/scripts/client/aircraft/AircraftModel.js]: I spy an oopsie 😄

File: src/assets/scripts/client/aircraft/AircraftModel.js [hunk 2]
```diff
@@ -2396,25 +2396,22 @@ export default class AircraftModel {
...
      * @param {AircraftConflict} conflict
-     * @param {Aircraft} conflictingAircraft
+     * @param {AircraftModel} conflictingAircraft
      */
     addConflict(conflict, conflictingAircraft) {
         this.conflicts[conflictingAircraft.callsign] = conflict;
     }
 
     /**
+     * Used to determine if a `conflictingAircraft` already exists within
```
Comment (erikquinn) [inline src/assets/scripts/client/aircraft/AircraftModel.js]: within what?

File: src/assets/scripts/client/aircraft/AircraftModel.js (hunk 1 above)
Comment (n8rzz) [reply src/assets/scripts/client/aircraft/AircraftModel.js]: ☹️

**Comment by n8rzz** [comment]
🤕

File: CHANGELOG.md [hunk 3]
```diff
@@ -18,9 +18,11 @@
 ### Enhancements & Refactors
 - [#956](https://github.com/openscope/openscope/issues/956) - Updated 'descend via STAR' documentation to change mentioning SID to STAR
 - [#782](https://github.com/openscope/openscope/issues/782) - Overhaul of KATL
+- [#916](https://github.com/openscope/openscope/issues/916) - consolidates aircraft removal logics, renames AircraftModel#inside_ctr -> AircraftModel#isControllable, simplifies aircraft update logic in AircraftController
 - [#961](https://github.com/openscope/openscope/issues/961) - Updated climb and descent rates using Eurocontrol data
 - [#874](https://github.com/openscope/openscope/issues/874) - Continue and clean up following FMS / Route refactor
 - [#982](https://github.com/openscope/openscope/issues/982) - Remove empty .gitkeep file
+- [#961](https://github.com/openscope/openscope/issues/961) - updated climb and descent rates using Eurocontrol data
```
Comment (erikquinn) [inline CHANGELOG.md]: Ah! Sorry, looks like one more issue, looks like 961 got duplicated here. The one capitalized on line 22 can stay, and this one on line 25 should be removed.

Comment (n8rzz) [reply CHANGELOG.md]: done

**Pull request merged**